### PR TITLE
Caduceus queue

### PR DESCRIPTION
**Pull request by gargidb**

### CODE REVIEW

File: outboundSender.go [hunk 1]
```diff
@@ -205,7 +211,13 @@ func (osf OutboundSenderFactory) New() (obs OutboundSender, err error) {
 
 	// Give us some head room so that we don't block when we get near the
 	// completely full point.
-	caduceusOutboundSender.queue = make(chan *wrp.Message, osf.QueueSize)
+	// insertQueue := make(chan *wrp.Message, osf.QueueSize)
+	// for i := 0; i < 10; i++ {
+	// 	caduceusOutboundSender.newQueue <- insertQueue
+	// }
+
+	// caduceusOutboundSender.queue = make(chan *wrp.Message, osf.QueueSize)
```
Comment (schmidtw) [inline outboundSender.go:219]: Go ahead and remove the old code vs. commenting it out.

File: outboundSender.go [hunk 2]
```diff
@@ -331,8 +343,8 @@ func (obs *CaduceusOutboundSender) Update(wh webhook.W) (err error) {
 // abruptly based on the gentle parameter.  If gentle is false, all queued
 // messages will be dropped without an attempt to send made.
 func (obs *CaduceusOutboundSender) Shutdown(gentle bool) {
-	close(obs.queue)
-
+	// close(obs.queue)
```
Comment (schmidtw) [inline outboundSender.go:346]: Go ahead and remove the old code vs. commenting it out.

File: outboundSender.go [hunk 3]
```diff
@@ -633,7 +635,7 @@ func (obs *CaduceusOutboundSender) queueOverflow() {
 		errorLog = logging.Error(obs.logger)
 	)
 
-	obs.queueEmpty = false
+	// obs.queueEmpty = false
```
Comment (schmidtw) [inline outboundSender.go:638]: Is the commented out part intentional (should it be removed)?

Comment (gargidb) [reply outboundSender.go:647]: I can remove it, it's being set a little earlier

**Pull request merged**